spec/controllers/api/public/v1/courses_controller_spec.rb

```diff
@@ -24,6 +24,20 @@
         provider.courses << build_list(:course, 2, provider:)
       end
 
+      context 'when "current" is specified as recruitment cycle' do
+        let!(:next_cycle) { create(:recruitment_cycle) }
```

Comment:
Is this required?

Reply:
no, it is not! Thanks for spotting that.